src/libraries/System.Text.Json/src/System/Text/Json/Writer/Utf8JsonWriter.cs

```diff
@@ -193,6 +193,34 @@ private ReadOnlySpan<char> PartialUtf16CodePoint
             }
         }
 
+        /// <summary>
+        /// The partial base64 code point.
```

Comment:
"Code point" is terminology more applicable to writing partial unicode characters, but probably not as much in the context of Base64 encodings. Presumably this represents a partial 3-byte block, so the name and comments should be renamed to reflect that.